### DISCUSSION TITLE
how can I set the activitypub subject field in my template to give a content warning?

### DISCUSSION THREAD

**Discussion by AlanBell** [Automattic/wordpress-activitypub#176] [General]
Mastodon appears to have subverted the subject field of the activitypub protocol as a content warning, which folds up the main text of the post. I would like to do that with blog posts, either use the subject of the post as the CW or some other text or field. Can I do that already somehow?

**Comment by pfefferle** [comment]
I will try to add a feature for that soon! Starting with a global field.

**Reply by caranha** [reply]
That would be a very appreciated feature!

**Reply by azizhp** [reply]
agreed, it would be very useful to be able to specify the post title as the CW. Looking forward to this feature release. I have the plugin enabled on my blog, haibane.info (user @otakun@www.haibane.info)
